### DISCUSSION TITLE
역할분담 및 차후 개발 관리

### DISCUSSION THREAD

**Discussion by myrhymetree** [goalapa/spring-cacamuca#14] [General]
1. 식품 -카테고리(주원)
2. 리뷰 & 좋아요(현석)
3. 신고, 블랙리스트(수의)
4. 회원(성준)
5. 요청 & 통계(정인)

개발은 백엔드 api부터 진행 - 퍼블리싱 - 프론트엔드 순서로.

github 이슈발행 및 pull request 규칙 정의
- 당일에 어떤 작업 할 것인지 다 같이 모여 논의하고, 이슈 발행하기(일단 점심시간 이후로)
- pull request는 이슈 하나 당 pr 한 번.
- pr 머지는 회의 진행 전 고정적으로 하는것.
- 작업이 덜 끝난 pr이여도 일단 날리고, 처리 못한 경우에는 close 하는것으로.

**Comment by bbbbooo** [comment]
확인했습니다~

**Comment by silverpoodle** [comment]
확인했습니다👍
